### PR TITLE
User admin app

### PR DESCRIPTION
**Pull request by ibrechin**
Allows for viewing and deleting users (placeholder UI awaiting design)

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include LICENSE
+include README.rst
+recursive-include mtp_user_admin/templates *
```
Comment (ushkarev) [inline MANIFEST.in]: maybe specify `*.html` so that .DS_Store and other rubbish doesn't get included

File: mtp_user_admin/views.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+from django.shortcuts import render
+from slumber.exceptions import HttpNotFoundError
+
+from moj_auth import api_client
+
+
+@login_required
+@permission_required('auth.change_user', raise_exception=True)
+def list_users(request):
+    users = api_client.get_connection(request).users.get()['results']
```
Comment (ushkarev) [inline mtp_user_admin/views.py]: is this resource paginated?

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,3 @@
+include LICENSE
```
Comment (ushkarev) [inline MANIFEST.in]: not attached

**Comment by ushkarev** [comment]
perhaps a simple user listing test with a mocked-out api?

File: MANIFEST.in (hunk 1 above)
Comment (ibrechin) [inline MANIFEST.in]: maybe don't use os x

Comment (ibrechin) [inline MANIFEST.in]: yeah i will change that though

File: setup.py [hunk 4]
```diff
@@ -13,6 +13,7 @@
 
 install_requires = [
     'Django>=1.9,<1.10',
+    'money-to-prisoners-utils==0.10'
```
Comment (ushkarev) [inline setup.py]: because the client apps also include this dependency, i'd suggest `>=0.10`

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+VERSION = importlib.import_module('mtp_user_admin').VERSION
+
+with open('README.rst') as readme:
+    README = readme.read()
+
+install_requires = [
+    'Django>=1.9,<1.10',
+    'money-to-prisoners-utils==0.10'
```
Comment (ushkarev) [inline setup.py]: because the client apps also include this dependency, i'd suggest `>=0.10`

**Pull request merged**